experimental/generated/csv/fields.csv

```diff
@@ -1263,203 +703,21 @@ ECS_Version,Indexed,Field_Set,Field,Type,Level,Normalization,Example,Description
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.domain,keyword,core,,foo.example.com,The domain name of the source.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.city_name,keyword,core,,Montreal,City name.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.continent_code,keyword,core,,NA,Continent code.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.continent_name,keyword,core,,North America,Name of the continent.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.country_iso_code,keyword,core,,CA,Country ISO code.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.country_name,keyword,core,,Canada,Country name.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.location,geo_point,core,,"{ ""lon"": -73.614830, ""lat"": 45.505918 }",Longitude and latitude.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.name,keyword,extended,,boston-dc,User-defined description of a location.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.postal_code,keyword,core,,94040,Postal code.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.region_iso_code,keyword,core,,CA-QC,Region ISO code.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.region_name,keyword,core,,Quebec,Region name.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.geo.timezone,keyword,core,,America/Argentina/Buenos_Aires,Time zone.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.ip,ip,core,,,IP address of the source.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.mac,keyword,core,,00-00-5E-00-53-23,MAC address of the source.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.nat.ip,ip,extended,,,Source NAT ip
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.nat.port,long,extended,,,Source NAT port
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.packets,long,core,,12,Packets sent from the source to the destination.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.port,long,core,,,Port of the source.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.registered_domain,keyword,extended,,example.com,"The highest registered source domain, stripped of the subdomain."
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.subdomain,keyword,extended,,east,The subdomain of the domain.
-8.3.0-dev+exp,true,process,process.parent.entry_meta.source.top_level_domain,keyword,extended,,co.uk,"The effective top level domain (com, org, net, co.uk)."
-8.3.0-dev+exp,true,process,process.parent.entry_meta.type,keyword,extended,,,The entry type for the entry session leader.
-8.3.0-dev+exp,true,process,process.parent.env_vars,object,extended,,"{""USER"": ""elastic"",""LANG"": ""en_US.UTF-8"",""HOME"": ""/home/elastic""}",Environment variables set at the time of the event.
```

Comment:
Another ? for `process.parent.env_vars`

I don't see any `env_vars` in https://github.com/elastic/endpoint-package/blob/49d9a9ac02762afa3ad158febe5edc164883d715/custom_subsets/elastic_endpoint/process/linux_event_model_event.yaml , but I know we added it for it, so just double checking

Reply:
I would say only top level process gets env_vars. I will submit another PR to endpoint-package.

Reply:
I added `process.parent.env_vars` to keep `process.*` and `process.parent.*` more symmetrical. Open to changing if there's any differing views.